src/__test__/index.test.ts

```diff
@@ -1,41 +1,272 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable import/first */
+// Some hackery to get WebSocket in the global namespace on nodejs
+// @todo: Find a nicer way to do this...
+;(global as any).WebSocket = require('isomorphic-ws')
+
 import { expect } from 'chai'
-import { NodeHttpTransport } from '@improbable-eng/grpc-web-node-http-transport/lib'
+import { NewStoreReply } from '@textile/threads-client-grpc/api_pb'
+import { WriteTransaction } from 'src/WriteTransaction'
 import { Client } from '../index'
 
-const client = new Client('http://localhost:9091', NodeHttpTransport())
+const sleep = (ms: number) => new Promise(r => setTimeout(r, ms))
+const client = new Client('http://localhost:9091')
 
-const personSchema = `{
-  "$id": "https://example.com/person.schema.json",
-  "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "Person",
-  "type": "object",
-  "properties": {
-    "firstName": {
-      "type": "string",
-      "description": "The person's first name."
+const personSchema = {
```

Comment:
Using an actual object here, as that is likely what callers will expect to to use (and it works 'as is')

Reply:
Nice, eventually we can do js -> schema. This lib here is nice and light: https://github.com/ruzicka/to-json-schema

Reply:
Ohh nice!